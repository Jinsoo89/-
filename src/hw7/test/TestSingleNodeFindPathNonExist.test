# Find a path with Dijkstra's algorithm when there is no possible path

# Create graph players, which has a single node
CreateGraph players
AddNode players C.Ronaldo

# Find path between the single node and non-exist node
FindPath players C.Ronaldo Z.Zidane
FindPath players P.Jisung L.Messi