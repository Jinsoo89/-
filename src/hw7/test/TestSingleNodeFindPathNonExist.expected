# Find a path with Dijkstra's algorithm when there is no possible path

# Create graph players, which has a single node
created graph players
added node C.Ronaldo to players

# Find path between the single node and non-exist node
unknown character Z.Zidane
unknown character P.Jisung
unknown character L.Messi